feature/gnmi/ate_tests/telemetry_basic_check_test/telemetry_basic_check_test.go

```diff
@@ -445,7 +478,12 @@ func TestSoftwareVersion(t *testing.T) {
 func TestCPU(t *testing.T) {
 	dut := ondatra.DUT(t, "dut")
 
-	r := regexp.MustCompile("^CPU")
+	supervisorMap := map[ondatra.Vendor]string{
+		ondatra.ARISTA: "^Supervisor[0-9]",
```

Comment:
Shouldnt this be "^CPU" ?

Reply:
There should be a lookup for a component with identity "CPU".  Supervisor is an Arista EOS name for an OpenConfig "CONTROLLER_CARD".

Reply:
Similar discussions in https://github.com/openconfig/featureprofiles/issues/829

Reply:
Changed to oc.PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT_CPU for CPU component